File: src/main/java/ash/java/graphql/types/tvseason/TvGuestStarType.java
package ash.java.graphql.types.tvseason;

import graphql.annotations.GraphQLField;
import graphql.annotations.GraphQLName;

@GraphQLName("TvGuestStar")
public class TvGuestStarType extends TvPersonType {

    @GraphQLField
    private String character;

    @GraphQLField
    private Integer order;

    public String getCharacter() {
        return character;
    }

    public void setCharacter(String character) {
        this.character = character;
    }

    public Integer getOrder() {
        return order;
    }

    public void setOrder(Integer order) {
        this.order = order;
    }
}

File: src/main/kotlin/ash/kotlin/graphql/types/tvseason/TvGuestStarType.kt
package ash.kotlin.graphql.types.tvseason

import graphql.annotations.*

@GraphQLName("TvGuestStar")
class TvGuestStarType(id: Int) : TvPersonType(id)
{

    @GraphQLField
    var character = ""

    @GraphQLField
    var order = 0
}
